Add `woocommerce-github-sync-labels` config.

tomalec opened the pull request:
### Changes proposed in this Pull Request:

- Add `woocommerce-github-sync-labels` config.
   So we could easily add labels used by https://github.com/woocommerce/grow/pull/38 to all our repos and somewhat maintain that list in future.

-  Fix `changelog: doc` label in `release.yml`

### Screenshots:

<!--- Optional --->


### Detailed test instructions:
<!-- Add detailed instructions for how to test that this PR fixes the issue and confirm that it doesn't break any other features :) -->

1. Follow instruction steps from the README, up to `dry-run`
2. Running actual command, could add labels to the repos, that may require manual delete.


### Additional details:

<!--
Optional.
Enter a summary of all changes in this Pull Request, which will be added to the changelog if accepted.
Each line should start with `(Fix|Add|Tweak|Update) - `, for example:
> Fix - I took care of something that wasn't working.
> Add - I added something new that's pretty cool.
> Tweak - I made a small change.
> Update - I made big changes to something that wasn't broken.

Leave the "Changelog entry" header in place completely empty, without any summary if no changelog entry is needed.
If you remove the "Changelog entry" header, the title of Pull Request will be used as the changelog entry.
-->
### Changelog entry

> Add - `woocommerce-github-sync-labels` config.

tomalec committed:
Add `woocommerce-github-sync-labels` config.

So we could easily add labels used by https://github.com/woocommerce/grow/pull/38 to all our repos and somewhat maintain that list.

tomalec committed:
Mention `-sync-labels` config in the main README

puntope commented on configs/woocommerce-github-sync-labels/README.md:
````diff
@@ -0,0 +1,18 @@
+
+1. [Install and setup `woocommerce-github-sync-labels`](https://github.com/woocommerce/woocommerce-github-sync-labels#install)
+2. Copy configs from this folder into the cloned tool folder 
+3. Run dry run
+	```
+	npm start -- --dry-run --preserve-labels
+	```
````
❓ Just curious, why we need to add `--dry-run` with `npm start`  ??

puntope replied:
Ok, I can see in https://github.com/woocommerce/woocommerce-github-sync-labels#install.

This is for testing? I think `npm test -- --preserve-labels` also works?

tomalec replied:
Yes, this is for testing. Given the operation is not easily reversible, I think the dry run/simulation is valuable.

puntope commented on configs/woocommerce-github-sync-labels/labels/commons.json:
```diff
@@ -0,0 +1,42 @@
+    "color": "5c7588",
+    "description": "Big changes to something that wasn't broken."
+  }
+]
```
💅  Missing newline at the end of the file

<img width="481" alt="Screenshot 2022-08-02 at 13 47 37" src="https://user-images.githubusercontent.com/5908855/182345660-bc69ddf2-73f4-4250-8249-aa4f6d9b7fb5.png">

puntope commented on configs/woocommerce-github-sync-labels/repositories.json:
```diff
@@ -0,0 +1,18 @@
+  "woocommerce/woocommerce-product-filters",
+  "woocommerce/woocommerce-product-finder",
+  "woocommerce/woocommerce-sales-report-email"
+]
```
💅 Missing newline at the end of the file

puntope reviewed (approved):
✅  LGTM

💅  Some issues to be addressed in regards EOF newlines. But approved in advance
 
- Tested locally with dry-run

```
➜  woocommerce-github-sync-labels git:(master) ✗ npm test --  --preserve-labels

> woocommerce-github-sync-labels@0.0.3 test
> node index.js --dry-run "--preserve-labels"


Starting sync...
✔️ "--dry-run" is enabled. No changes will happen in any repository.
✔️ "--preserve-labels" is enabled. New labels introduced directly in GitHub will be preserved.

Repositories:
  -woocommerce/automatewoo
  -woocommerce/automatewoo-birthdays
  -woocommerce/automatewoo-referrals
  -woocommerce/facebook-for-woocommerce
  -woocommerce/google-listings-and-ads
  -woocommerce/pinterest-for-woocommerce
  -woocommerce/woocommerce-brands
  -woocommerce/woocommerce-coupon-campaigns
  -woocommerce/woocommerce-drip
  -woocommerce/woocommerce-follow-up-emails
  -woocommerce/woocommerce-give-products
  -woocommerce/woocommerce-google-analytics-integration
  -woocommerce/woocommerce-points-and-rewards
  -woocommerce/woocommerce-product-filters
  -woocommerce/woocommerce-product-finder
  -woocommerce/woocommerce-sales-report-email
Labels synced for "woocommerce/woocommerce-brands"
Labels synced for "woocommerce/woocommerce-sales-report-email"
Labels synced for "woocommerce/woocommerce-google-analytics-integration"
Labels synced for "woocommerce/automatewoo"
Labels synced for "woocommerce/automatewoo-referrals"
Labels synced for "woocommerce/woocommerce-drip"
Labels synced for "woocommerce/woocommerce-product-filters"
Labels synced for "woocommerce/facebook-for-woocommerce"
Labels synced for "woocommerce/woocommerce-give-products"
Labels synced for "woocommerce/woocommerce-product-finder"
Labels synced for "woocommerce/woocommerce-points-and-rewards"
Labels synced for "woocommerce/pinterest-for-woocommerce"
Labels synced for "woocommerce/automatewoo-birthdays"
Labels synced for "woocommerce/woocommerce-coupon-campaigns"
Labels synced for "woocommerce/woocommerce-follow-up-emails"
Labels synced for "woocommerce/google-listings-and-ads"
```

tomalec committed:
Fix typo `doc~s~` typo in release.yml

Addresses https://github.com/woocommerce/pinterest-for-woocommerce/pull/505#discussion_r937611567

Co-authored-by: Miguel Perez Pellicer <5908855+puntope@users.noreply.github.com>

tomalec commented:
@puntope I added your fix from https://github.com/woocommerce/pinterest-for-woocommerce/pull/505#discussion_r937611567 to here as well, to keep the label names in sync between release and labels configs.

tomalec committed:
Add missing line breaks at EOF

Addresses https://github.com/woocommerce/grow/pull/39#pullrequestreview-1058467552

puntope reviewed (approved):
Thanks for the tweak! LGTM
